pkg/cloudprovider/aws/apis/v1alpha1/register.go

```diff
@@ -32,6 +32,8 @@ var (
 	AWSRestrictedLabelDomains = []string{
 		"k8s.aws",
 	}
+	OperatingSystemBottleRocket = "BottleRocket"
```

Comment:
per the project, it should be one word: `bottlerocket` https://github.com/bottlerocket-os/bottlerocket#bottlerocket-os

Reply:
I had some feedback here. It should be upper case since it's a yaml value. `Bottlerocket`.

Reply:
Fixed - my bad.